README.md

```diff
@@ -2,6 +2,9 @@
 
 We love pull requests from everyone. This repository is created for you to contribute different algorithms in different languages. There are separate folders for different languages.
 
+## Edit
```

Comment:
There is a section to add your algorithm in the readme. Please add it there, attaching screenshot for your reference. 
![Screenshot from 2020-10-01 12-06-14](https://user-images.githubusercontent.com/26180236/94776083-a91ff900-03de-11eb-8c95-eb2163692980.png)